profiles/Scaleway (PL-WAW).cyberduckprofile

```diff
@@ -0,0 +1,48 @@
+<!--
+  ~ Copyright (c) 2002-2018 iterate GmbH. All rights reserved.
+  ~ https://cyberduck.io/
+  ~
+  ~ This program is free software; you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation, either version 3 of the License, or
+  ~ (at your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  -->
+
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>Protocol</key>
+        <string>s3</string>
+        <key>Vendor</key>
+        <string>scaleway-pl-waw</string>
+        <key>Description</key>
+        <string>Scaleway Object Storage (PL-WAW)</string>
+        <key>Default Hostname</key>
+        <string>s3.pl-waw.scw.cloud</string>
+        <key>Default Port</key>
+        <string>443</string>
+        <key>Username Placeholder</key>
+        <string>Access Key</string>
+        <key>Password Placeholder</key>
+        <string>Secret Key</string>
+        <key>Authorization</key>
+        <string>AWS4HMACSHA256</string>
+        <key>Region</key>
+        <string>pl-waw</string>
+        <key>Regions</key>
+        <array>
+            <string>pl-waw</string>
+        </array>
+        <key>Properties</key>
+        <array>
+            <string>s3.storage.class.options=STANDARD GLACIER</string>
+        </array>
+        <key>Help</key>
+        <string>https://trac.cyberduck.io/wiki/help/en/howto/scaleway</string>
```

Comment:
Would you mind updating the help string as well?
```suggestion
        <string>https://docs.cyberduck.io/protocols/s3/scaleway/</string>
```

Reply:
Sure, done on the 3 profiles.

Thanks for letting me know!